homeassistant/components/keyboard_remote.py

```diff
@@ -40,7 +41,15 @@
         vol.Exclusive(DEVICE_DESCRIPTOR, DEVICE_ID_GROUP): cv.string,
         vol.Exclusive(DEVICE_NAME, DEVICE_ID_GROUP): cv.string,
         vol.Optional(TYPE, default='key_up'):
-        vol.All(cv.string, vol.Any('key_up', 'key_down', 'key_hold')),
+            vol.All(cv.string, vol.Any('key_up', 'key_down', 'key_hold')),
+        vol.Exclusive(DEVICES, DEVICE_ID_GROUP): vol.All(cv.ensure_list, [
```

Comment:
Instead of adding `ensure_list` here, you can just wrap the whole DOMAIN config as an ensure list. No need to specify the schema twice.

Reply:
I have done so. Be aware this means you can never backwards-compatibly add module-level (not keyboard-level) configuration options in the future.